.travis.yml

```diff
@@ -8,10 +8,6 @@ dist: trusty
 
 bundler_args: --without development
 
-branches:
-  only:
-    - master
-
```

Comment:
@varyonic sometimes likes to run CI on a branch without opening a PR. With our current Travis CI settings, this is the only way to do it.